index.html

```diff
@@ -1,204 +1,186 @@
-        <div class="section-info">
-          <h1 class="title">Hey There.<br> I'm Samuel</h1>
-          <h4 class="sub-title">I'am a software Developer</h4>
-          <p class="paragraph">
-            I can help you build a product , feature or website Look through
-            some of my work and experience! If you like what you see and have a
-            project you need coded, don't hestiate to contact me.
+
+<head>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <link rel="stylesheet" href="index.css" />
+  <title>Portfolio</title>
+</head>
+
+<body>
+  <header class="header">
+    <nav class="navbar">
+      <a class="logo" href="#">My Logo</a>
+      <button type="button" class="hamburger-button">&#8801;</button>
+      <div class="navbar-links">
+        <ul>
+          <li>Portfolio</li>
+          <li>About</li>
+          <li>Contact</li>
+        </ul>
+        <button type="button" class="mail">&#9993;</button> 
+      </div>
+    </nav>
+  </header>
+  <section class="section1">
+    <div class="section-info">
+      <h1 class="title">Hey There.<br> I'm Samuel</h1>
+      <h4 class="sub-title">I'am a software Developer</h4>
+      <p class="paragraph">
+        I can help you build a product , feature or website Look through
+        some of my work and experience! If you like what you see and have a
+        project you need coded, don't hestiate to contact me.
+      </p>
+    </div>
+      <ul class="items">
+        <li>
+          <a href="#"><img src="images/Github1.png" alt="github" /></a>
+        </li>
+        <li>
+          <a href="#"><img src="images/Inner1.png" alt="images-iner" /></a>
+        </li>
+        <li>
+          <a href="#"><img src="images/Peace1.png" alt="media" /></a>
+        </li>
+        <li>
+          <a href="#"><img src="images/Twitter1.png" alt="twitter-icon" /></a>
+        </li>
+        <li>
+          <a href="#"><img src="images/Mail1.png" alt="mail-icon" /></a>
+        </li>
+      </ul>
+  </section>
+  <section class="section2">
+    <div class="section2-Head_lining">
+      <h2>My Recent Works</h2>
+      <hr />
+    </div>
+    <div class="section2-placeholder">
+      <img class="placeholder" src="images/Placeholder.png" alt="placeholder" />
+      <div class="project-info">
+        <div class="section2-wordings">
+          <h3 class="stories">Multi-Post Stories</h3>
+          <p class="section2-paragraph">
+            A daily selection of privately personalized reads; no accounts
+            or sign-ups required. has been the industry's standard dummy
+            text ever since the 1500s, when an unknown printer took a
+            standard dummy text.
           </p>
+          <div class="language-container">
+            <p>CSS</p>
+            <p>HTML</p>
+            <p>Bootstrap</p>
+            <p>Ruby</p>
+          </div>
```

Comment:
- [ ] Please wrap these listings into `<ul> <li> </li> </ul>` tags  like you did it in other sections 😉